src/getCoinCombination.test.js

```diff
@@ -1,10 +1,16 @@
 'use strict';
 
-describe('getCoinCombination', () => {
-  const { getCoinCombination } = require('./getCoinCombination');
+const { getCoinCombination } = require('./getCoinCombination');
 
+describe('getCoinCombination', () => {
   it(`should be declared`, () => {
-    expect(getCoinCombination)
-      .toBeInstanceOf(Function);
+    expect(getCoinCombination).toBeInstanceOf(Function);
+  });
+
+  it('should convert cents to coin combination correctly', () => {
+    expect(getCoinCombination(1)).toEqual([1, 0, 0, 0]);
+    expect(getCoinCombination(6)).toEqual([1, 1, 0, 0]);
+    expect(getCoinCombination(17)).toEqual([2, 1, 1, 0]);
+    expect(getCoinCombination(50)).toEqual([0, 0, 0, 2]);
   });
```

Comment:
It would be better to separate these expects to different tests. Also not all required tests were written, for example what if the function takes 0?) with which value it should return [1,1,1,1] etc? Add more tests